app/models/create_cap_proposal.rb

```diff
@@ -0,0 +1,31 @@
+class CreateCapProposal
+  def initialize(auction)
+    @host = ENV.fetch('C2_HOST', 'https://cap.18f.gov')
+    @c2_client = C2::Client.new(
```

Comment:
I wonder if we should move this into an initializer so it'll blow up immediately if those ENV vars are not found (right now it'll just error out the jobs). But I might be overthinking this.